### PR TITLE
Fix template-base-3-typescript typescript error

### PR DESCRIPTION
**Pull request by zbigg**
Skip import of @deck.gl in typescript code.

### CODE REVIEW

File: template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { useDispatch } from 'react-redux';
 import { makeStyles } from '@material-ui/core';
 import { setViewState, ViewState } from '@carto/react-redux';
-import { PickInfo } from 'deck.gl';
```
Comment (zbigg) [inline template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: Alternative solution would be to add unofficial [@deck.gl typings](https://www.npmjs.com/package/@danmarshall/deckgl-typings), but their usage is not-as-simple as we would like.

Opinions?

Comment (VictorVelarde) [reply template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: I wouldn't add that as a requirement now. Especially as there is WIP in deck.gl to have TS. Thoughts @Clebal ?

Comment (Clebal) [reply template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: Try to import PickInfo from `@deck.gl/core/lib/deck`. Using `from 'deck.gl'`, I think it tries to import it from `node_modules/@types/deck.gl` but for some reason that idk it doesn't work after installing some package.

Comment (zbigg) [reply template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: @Clebal 

> @deck.gl/core/lib/deck

It doesn't work, again
> Cannot find module '@deck.gl/core/lib/deck' or its corresponding type declarations.

deck.gl _doesn't_ provide typings. Typescript can't work pr

```
$ ls -al node_modules/@types/deck.gl
ls: node_modules/@types/deck.gl: No such file or directory
```

Comment (Clebal) [reply template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: Try to remove node_modules and `yarn install` again.

Comment (VictorVelarde) [reply template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: Let's go for the simplest solution for now, which is zbig's. We must do release. And we can explorer TS improvements after 1.2

Comment (Clebal) [reply template-base-3-typescript/template/src/components/common/map/useMapHooks.tsx:4]: Agree

**Review by Clebal — APPROVED** [review]

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Pull request merged**